spec/helper.js

```diff
@@ -21,6 +21,27 @@ global.onHttpContext = index.onHttpContextFactory();
 // Legacy
 global.dihelper = onHttpContext.helper;
 
+helper.httpServerBefore = function(overrides, endpointOpt) {
+    before('helper.httpServer.before', function() {
+        this.timeout(5000);
+        return helper.startServer(overrides, endpointOpt);
+    });
+
+    beforeEach(function () {
```

Comment:
Since on-core helper.js  has already create the sandbox, my suggestion is directly calling on-core's `before` (https://github.com/RackHD/on-core/blob/master/spec/helper.js#L268) rather than writing some duplicated code.